x-pack/plugins/apm/server/ui_settings.ts

```diff
@@ -28,4 +31,18 @@ export const uiSettings: Record<string, UiSettingsParams<boolean>> = {
     ),
     schema: schema.boolean(),
   },
+  [enableDebugQueries]: {
```

Comment:
Is this available to all users? If so, are there any security concerns we need to take into account here? e.g. data from calls from the internal user?

Reply:
Good point, I've already removed this from requests made by the internal user since that's definitely a security concern.
But apart from that this should be fine. I asked the security folks a while back and they gave the green light as long as we only output requests made by the end user.